src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -180,26 +200,125 @@ public void perform(
     // generate a kubeconfig for the cluster
     KubeConfig kubeConfig = KubeConfig.fromCluster(projectId, cluster);
 
-    // run kubectl apply
-    KubectlWrapper.runKubectlCommand(
+    JenkinsRunContext context =
         new JenkinsRunContext.Builder()
             .workspace(workspace)
             .launcher(launcher)
             .taskListener(listener)
             .run(run)
-            .build(),
+            .build();
+    KubectlWrapper kubectl = new KubectlWrapper(context, kubeConfig);
+    kubectl.runKubectlCommand(
+        context,
         kubeConfig,
         "apply",
-        ImmutableList.<String>of("-f", workspace.child(manifestPattern).getRemote()));
+        ImmutableList.<String>of("-f", workspace.child(manifestPattern).getRemote(), "-o", "yaml"));
```

Comment:
If the output of the initial deploy command isn't captured here, is it necessary to append -o yaml?

Reply:
No I was using that previously. Will remove.